### ISSUE TITLE
/custom-name has serious bugs.

### ISSUE THREAD

**Issue by ub0nFPS** [codehz/ElementZero#68]
When naming someones prefix or postfix with /custom-name you can no longer /tp to them or execute @ them in any way.

The only way to fix this is to clear their name.
No idea when this will be fixed.

**Comment by ambiennt** [comment]
> When naming someones prefix or postfix with /custom-name you can no longer /tp to them or execute @ them in any way.
> 
> The only way to fix this is to clear their name.
> No idea when this will be fixed.

yes, I do agree that the custom name has serious problems. When executing commands on behalf of all players or selectors that do not directly use the player's name (when he or she has a custom name), their custom name clears and is reset to the default. im not sure why this happens.
